## D2 compatible `onRangeError` definition

New `extern(C)` function was added to `core.exception` module - `onRangeError`.
it simply calls existing `onArrayBoundsError` function. It was added to make API
more compatible with D2 runtime which doesn't have `onArrayBoundsError`.
